phoenix-core/src/main/java/org/apache/phoenix/coprocessor/tasks/IndexRebuildTask.java

```diff
@@ -162,7 +163,9 @@ private String getJobID(String data) {
             Cluster cluster = new Cluster(configuration);
 
             Job job = cluster.getJob(org.apache.hadoop.mapreduce.JobID.forName(jobID));
-
+            if (job == null) {
```

Comment:
@gokceni where is this used?

Reply:
When you kick the task and it runs index tool, cluster.getJob doesn't return a job immediately sometimes. We have to come back and requery the job status when we find the job from cluster.getJob